pkg/runtime/containerd/client.go

```diff
@@ -28,19 +30,32 @@ import (
 	log "github.com/sirupsen/logrus"
 	meta "github.com/weaveworks/ignite/pkg/apis/meta/v1alpha1"
 	"github.com/weaveworks/ignite/pkg/preflight"
-	"github.com/weaveworks/ignite/pkg/preflight/checkers"
 	"github.com/weaveworks/ignite/pkg/runtime"
 	"github.com/weaveworks/ignite/pkg/util"
 	"golang.org/x/sys/unix"
 )
 
 const (
-	ctdSocket        = "/run/containerd/containerd.sock"
 	ctdNamespace     = "firecracker"
 	stopTimeoutLabel = "IgniteStopTimeout"
 	logPathTemplate  = "/tmp/%s.log"
 )
 
+var (
+	// containerdSocketLocations is a list of socket locations to stat for
+	containerdSocketLocations = []string{
```

Comment:
This is good that we could cope with various versions of Docker!

Reply:
Can you confirm that building this works for your containerd socket location?
I tried it out in container by touching the file, and it seemed to work.